docs/data/routes/docs/nextjs/tracking-and-analytics/overview/en.md

```diff
@@ -5,11 +5,11 @@ title: Overview and architecture
 ---
 # Tracking & Analytics Overview and Architecture
```

Comment:
Having "&" and "and" in the same line feels odd

```suggestion
# Overview and Architecture of Tracking and Analytics
```

Reply:
Will defer to @AncaIO  :)

Reply:
@ambrauer Wouldn't "Tracking & Analytics Conceptual Overview" work better here? Technically, you are explaining what links JSS and Sitecore for Tracking & Analytics via the "Header passing" concept AND explaining the diff between client-side tracking and Sitecore T&A.

Reply:
@AncaIO I'm fine with that, though the Header Passing bit is more of a deep dive and feels more architecture than conceptual. We could also just switch the "&" to "and" like we do on other pages (https://jss.sitecore.com/docs/nextjs/data-fetching/getStaticPaths) - where "&" is reserved for the nav only.

Reply:
@ambrauer Why are those headers relevant? Could you provide a brief description for each and why they are needed? 
What cookies are we getting back? 
I am looking for more concrete information about the values passed back and forth for the purposes of cross-origin requests, tracking permission cookies or anything similar that we might need to document / inform about and/or link to the main Sitecore docs.

Reply:
@AncaIO Here's a brief description of each. Hope this helps?

- **cookie/set-cookie** - Sitecore's session tracking is cookie-based, see https://doc.sitecore.com/developers/101/platform-administration-and-architecture/en/cookies-used-by-sitecore.html (Sitecore visitor cookies) for details. 'SC_ANALYTICS_GLOBAL_COOKIE' is the main analytics cookie.
- **user-agent** - Used by Sitecore Analytics for reporting, robot detection and more (should be able to find relevant docs on doc.sitecore.com).
- **referer** - Used by Sitecore Analytics for reporting, to determine the traffic type and more (should be able to find relevant docs on doc.sitecore.com).
- **X-Forwarded-For** - User's IP address (https://nodejs.org/api/net.html#net_socket_remoteaddress), see https://jss.sitecore.com/docs/nextjs/tracking-and-analytics/configuration#forwarded-request-header for related JSS doc. Used by Sitecore Analytics for reporting, personalization, robot detection and more (should be able to find relevant docs on doc.sitecore.com).